package.json

```diff
@@ -35,8 +36,11 @@
     }
   },
   "dependencies": {
+    "@fastify/express": "^1.1.0",
     "express": "^4.17.3",
     "got": "^11.5.2",
-    "require-in-the-middle": "^5.1.0"
+    "require-in-the-middle": "^5.1.0",
+    "fastify": "^3.29.0",
```

Comment:
Why are we adding **fastify** and **pino**?

Reply:
Oh forgot to delete it